Sources/MotorControl/Model/MotionSensorNodeObject.swift

```diff
@@ -32,18 +32,21 @@ class TremorNodeObject : MotionSensorNodeObject {
     }
 }
 
-class WalkNodeObject : MotionSensorNodeObject {
+class WalkNodeObject : WalkOrBalanceNodeObject {
     override class func defaultType() -> SerializableNodeType {
         .walk
     }
 }
 
-class BalanceNodeObject : MotionSensorNodeObject {
+class BalanceNodeObject : WalkOrBalanceNodeObject {
     override class func defaultType() -> SerializableNodeType {
         .balance
     }
 }
 
+class WalkOrBalanceNodeObject : MotionSensorNodeObject {
+}
```

Comment:
Feels a bit more like where you might use a flag (or protocol or generic) that than checking cast and having these be different types?